### PR TITLE
Add renovate configuration

### PR DESCRIPTION
**Pull request by dhaiducek**
Set to update only on weekends

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "schedule": "0 0 * * 1",
```
Comment (JustinKuli) [inline .github/renovate.json:3]: From https://docs.renovatebot.com/configuration-options/#schedule
> Warning
> For Cron schedules, you must use the * wildcard for the minutes value, as Renovate doesn't support minute granularity.

Comment (dhaiducek) [reply .github/renovate.json]: At least _one_ of us is reading the docs 😆

**Review by JustinKuli — APPROVED** [review]
Worth a shot!

**Pull request merged**